intelmq/lib/harmonization.py

```diff
@@ -856,10 +856,10 @@ def sanitize(value: Union[int, str]) -> Optional[str]:
     def to_int(value: str) -> Optional[int]:
         try:
             ip_integer = socket.inet_pton(socket.AF_INET, value)
-        except socket.error:
+        except OSError:
             try:
                 ip_integer = socket.inet_pton(socket.AF_INET6, value)
-            except socket.error:
+            except OSError:
```

Comment:
Still confused about the OSError instead of socket.error

Reply:
https://docs.python.org/3/library/socket.html?highlight=socket%20error#socket.error
> A deprecated alias of OSError.